### PR TITLE
[api] Fixed API route response

### PR DESCRIPTION
**Pull request by assistechsol**
This PR:
- updates the API output, to return the text for each page

### CODE REVIEW

File: api/app/routes/ocr.py [hunk 1]
```diff
@@ -21,4 +21,4 @@ async def perform_ocr(file: UploadFile = File(...)):
     out = predictor([img])
 
     return [OCROut(box=(*word.geometry[0], *word.geometry[1]), value=word.value)
-            for word in out.pages[0].blocks[0].lines[0].words]
+            for index in range(len(out.pages[0].blocks)) for word in out.pages[0].blocks[index].lines[0].words]
```
Comment (frgfm) [inline api/app/routes/ocr.py:24]: This piece of code is still only checking the first page only and only the first line of each block. The route is single-page only for now, so let's fix this :) I would suggest:
```python
for block in out.pages[0].blocks for line in block.lines for word in line.words
```

Comment (frgfm) [reply api/app/routes/ocr.py:24]: Would you mind updating the PR please? 🙏

**Review by frgfm — CHANGES_REQUESTED** [review]
Hello @assistechsol 👋 

Thanks a lot for spotting this and opening the PR! I added a slight correction suggestion as comment!

Cheers!

File: api/app/routes/ocr.py (hunk 1 above)
Comment (frgfm) [reply api/app/routes/ocr.py:24]: Yes but this route is single page for now!
So this means that pages only have 1 element 👍

Comment (frgfm) [reply api/app/routes/ocr.py:24]: Hello @SamSamhuns @assistechsol :wave: 

What do you think about my previous comment?
Let's find a solution to move forward with this PR :+1:

Comment (SamSamhuns) [reply api/app/routes/ocr.py:24]: Sounds good, probably a good idea to mention somewhere its only for one page tho

Comment (frgfm) [reply api/app/routes/ocr.py:24]: Yes! Mind updating the PR then? :pray:

**Comment by felixdittrich92** [comment]
Hi @assistechsol :wave:, any updates ? :)

**Comment by felixdittrich92** [comment]
closed with: no response

**Pull request closed without merging**